rental.rb

```diff
@@ -1,6 +1,5 @@
 class Rental
-  attr_reader :book, :person
-  attr_accessor :date
+  attr_accessor :date, :book, :person
 
   def initialize(date, book, person)
     @date = date
```

Comment:
- [ ] When I try to create a new rental there is an error raised (See the image below), So please fix that.

![image](https://user-images.githubusercontent.com/68796232/200126832-b50249d8-aea2-4e6d-a372-52f54e68bf9a.png)

Reply:
Well resolved. Rental created successfully 👍🏽